### PR TITLE
Add bancornetwork to dex.trades

### PR DESCRIPTION
**Pull request by mewwts**

### CODE REVIEW

File: schema/dex/trades.sql [hunk 1]
```diff
@@ -446,7 +446,26 @@ WITH rows AS (
...
-        FROM gnosis_protocol."view_trades"
+        FROM gnosis_protocol.view_trades
+
+        UNION
+
+        -- Bancor Network
+        SELECT
+			block_time,
+			'Bancor Network' AS project,
+			NULL AS version,
```
Comment (mewwts) [inline schema/dex/trades.sql:457]: Used NULL here as the view contains all versions

File: schema/dex/trades.sql [hunk 2]
```diff
@@ -446,7 +446,26 @@ WITH rows AS (
...
-        FROM gnosis_protocol."view_trades"
+        FROM gnosis_protocol.view_trades
+
+        UNION
+
+        -- Bancor Network
+        SELECT
+            block_time,
+            'Bancor Network' AS project,
+            NULL AS version,
```
Comment (mewwts) [inline schema/dex/trades.sql:457]: Used NULL as version here as view selects from multiple versions

**Comment by mewwts** [comment]
@milkyklim WDYT

**Comment by milkyklim** [comment]
LGTM!

File: schema/dex/trades.sql [hunk 3]
```diff
@@ -446,7 +446,26 @@ WITH rows AS (
...
+            trader AS trader_a,
+            NULL::bytea AS trader_b,
+            target_token_amount_raw AS token_a_amount_raw,
+            source_token_amount_raw AS token_b_amount_raw,
+            target_token_address AS token_a_address,
+            source_token_address AS token_b_address,
```
Comment (milkyklim) [inline schema/dex/trades.sql:463]: Just realized that these might be wrong: should not `token_a` be the `source` and `token_b` be the `target`?

Comment (mewwts) [reply schema/dex/trades.sql:463]: @hagaetc wdyt?

**Comment by mewwts** [comment]
Close in favor of #82

**Pull request closed without merging**